### PR TITLE
doc: document how to keep config updated

### PR DESCRIPTION
**Pull request by acateland**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ It will do the following:
 * Setup a default .gitignore file and make the initial git commit
 * Run the initial `npm install`
 
+Run the `yo talend-lib` command again whenever you want to make sure configuration files, license are updated
```
Comment (frassinier) [inline README.md]: Run the `yo talend-lib` command again **when** you want to make sure configuration files and license are **up to date**

**Comment by sgandon** [comment]
Could it be possible to add more details on the *configuration files* that are installed from the **tools** repository ?

**Comment by talend-glorieux** [comment]
@sgandon Will do ;)

**Comment by talend-glorieux** [comment]
Will have to be merged after #7.

**Pull request merged**